client/src/pages/Messages.tsx

```diff
@@ -294,9 +326,11 @@ const Profile = () => {
 
       if (user) {
         sendMessage({
-          action: '@@MESSAGES/SEND',
+          action: '@@MESSAGES/SAVE',
```

Comment:
We are asking the system to _save_ a message and assuming it will do the needful for _sending_ it eventually